pkg/datastore/datastore_test.go

```diff
@@ -1,26 +1,36 @@
-package datastore_test
+package datastore
 
 import (
 	"strings"
 	"testing"
 
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-
-	"github.com/operator-framework/operator-marketplace/pkg/appregistry"
-	"github.com/operator-framework/operator-marketplace/pkg/datastore"
 )
 
 func TestGetPackageIDs(t *testing.T) {
+	controller := gomock.NewController(t)
+	defer controller.Finish()
+
 	expected := []string{"foo/1", "bar/2", "braz/3"}
 
-	packages := []*appregistry.OperatorMetadata{
-		&appregistry.OperatorMetadata{Namespace: "foo", Repository: "1"},
-		&appregistry.OperatorMetadata{Namespace: "bar", Repository: "2"},
-		&appregistry.OperatorMetadata{Namespace: "braz", Repository: "3"},
+	packages := []*OperatorMetadata{
+		helperNewOperatorMetadata("foo", "1"),
+		helperNewOperatorMetadata("bar", "2"),
+		helperNewOperatorMetadata("braz", "3"),
```

Comment:
Minor nit: Any reason behind `braz` here? I would have expected `foobar`

Reply:
Actually, this is a typo. It should be `baz`. 
From Wikipedia - `foo, bar, baz, qux, quux, quuz, corge, grault, garply, waldo, fred, plugh, xyzzy, and thud.[1][3] Wibble, wobble, wubble, and flob are also used in the UK.` :)

You are welcome to fix the typo in a separate PR :)

Reply:
Actually, I like the sound of `braz` over something like `garply`